nav2_bringup/launch/nav2_bringup_1st_launch.py

```diff
@@ -0,0 +1,32 @@
+import os
+from launch import LaunchDescription
+import launch.actions
+import launch_ros.actions
+
+def generate_launch_description():
+    map_file = launch.substitutions.LaunchConfiguration('map')
+    map_type = launch.substitutions.LaunchConfiguration('map_type', default='occupancy')
+    use_sim_time = launch.substitutions.LaunchConfiguration('use_sim_time', default='false')
+
+    return LaunchDescription([
+        launch.actions.DeclareLaunchArgument(
+            'map', description='Full path to map file to load'),
+        launch.actions.DeclareLaunchArgument(
+            'map_type', default_value='occupancy', description='Type of map to load'),
+        launch.actions.DeclareLaunchArgument(
+            'use_sim_time', default_value='false', description='Use simulation (Gazebo) clock if true'),
+
+        launch_ros.actions.Node(
+            package='nav2_map_server',
+            node_executable='map_server',
+            output='screen',
+            arguments=[map_file, map_type]),
+
+        launch_ros.actions.Node(
+            package='nav2_amcl',
+            node_executable='amcl',
+            node_name='amcl',
+            output='screen',
+            parameters=[{ 'use_sim_time': use_sim_time}]),
```

Comment:
We figured out that the `use_sim_time` parameter doesn't work, if I recall. You need to publish a parameter event after the node is up and listening.

We should just remove these and add another section to the readme saying run `ros2 param set /gazebo use_sim_time true`

Reply:
@crdelsey - the use_sim_time does get set, but not for the transform, I've updated the readme to add that. Please look at the last commit.

Reply:
Works for AMCL, but I'm getting errors now for world_model. I need to trigger some param update for the warning to stop.

`WARN global_costmap: Costmap2DROS transform timeout. Current time: 1543355603.4700, global_pose stamp: 160.2020, tolerance: 0.3000, difference: 1543355443.2680`